test/providerLoader/test.html

```diff
@@ -0,0 +1,21 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <meta charset="utf-8" />
+        <title>Test Runner - Runner Component</title>
```

Comment:
Wrong title, should be `Test Runner - Provider Loader`

Reply:
I just copied the file, but I totally agree. Fixed.